validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+| 11 | Activate DR volume in target cluster | | DR volume activation succeeds |
+| 12 | Deploy application using activated DR volume | | Application successfully uses DR volume |
+| 13 | Verify data integrity matches source cluster | | All test data present and correct |
+
+### Node Drain Operation via Rancher
+
+Test graceful node drain functionality with Longhorn volumes ensuring proper replica evacuation and data integrity.
+
+| Step Number | Action | Data | Expected Result |
+|------|--------|--------|-----------------|
+| 1 | Identify node with attached Longhorn volumes | | Target node selected with active volumes |
+| 2 | Document current volume replica distribution | | Baseline replica placement recorded |
+| 3 | Navigate to "Nodes" in Rancher cluster view | | Nodes section displays |
+| 4 | Select target node and click "..." → "Cordon" | | Node cordoned successfully |
+| 5 | Click "..." → "Drain" with grace period 300s | | Drain operation begins |
+| 6 | Monitor drain progress in Rancher UI | | Drain proceeds respecting PDB constraints |
+| 7 | Verify Longhorn volume status remains healthy during drain | | No volume degradation occurs |
+| 8 | Check Longhorn UI for replica evacuation | | Replicas evacuate before pod termination |
+| 9 | Confirm drain completes successfully | | Node shows as drained and cordoned |
+| 10 | Verify all volumes maintain healthy status | | All volumes remain in "Robust" state |
+| 11 | Test uncordon operation by clicking "..." → "Uncordon" | | Node returns to schedulable state |
+
+### Node Scaling with Auto-Configuration
+
+Test cluster scaling operations with automatic Longhorn node configuration and replica distribution.
+
+| Step Number | Action | Data | Expected Result                    |
+|------|--------|--------|-------------------------------------|
+| 1 | Document baseline: current node count and storage capacity | | Baseline metrics recorded           |
+| 2 | Navigate to "Nodes" → "Add Node" in Rancher | | Node addition interface opens       |
+| 3 | Add 2 new nodes following Rancher registration process | | Nodes join cluster successfully     |
+| 4 | Verify new nodes reach "Active" state in Rancher | | Nodes show as ready and active      |
+| 5 | Check Longhorn UI for automatic node configuration | | New nodes appear with default disks |
+| 6 | Create new volumes to trigger replica placement | | Volumes created across all nodes    |
+| 7 | Verify replica distribution includes new nodes | | Replicas distributed to new nodes   |
+| 8 | Select node for removal and verify volume replicas on other nodes | | Target node has movable replicas    |
+| 9 | Drain selected node using Rancher drain functionality | | Node drained successfully           |
+| 10 | Remove node from cluster | | Node removal completes              |
+| 11 | Verify volume health maintained after node removal | | All volumes remain healthy          |
+
+### Longhorn Upgrade via Rancher Apps
```

Comment:
would rename to just `Upgrade` or maybe `Upgrade Chart`

Reply:
Why

Reply:
I thought these are only rancher tests, so upgrading through rancher apps would be implied.